examples/bert/run_glue_finetuning.py

```diff
@@ -167,9 +162,27 @@ def call(self, inputs):
         return self._logit_layer(outputs)
 
 
+class BertHyperModel(kt.HyperModel):
+    """Creates a hypermodel to help with the search space for finetuning."""
+
+    def __init__(self, model):
+        self.model = model
+
+    def build(self, hp):
+        model = self.model
```

Comment:
Passing a model to the `HyperModel` may cause problems. The model should be recreated everytime `.build()` is called. So prefer changing it to `model=BertClassificationFinetuner(...)`. And the `__init__` can be removed.

Reply:
I didn't know about this, I'll make the requested changes. It's just that there was an issue while using `BertClassficationFinetuner(...)` that I won't be able to access variables like `bert_config` which are local. So, I guess we can change this into a local class, where we'll be able to access those variables.